pkgs/by-name/fu/fusion/package.nix

```diff
@@ -0,0 +1,67 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildNpmPackage,
+  buildGoModule,
+  mockgen,
+  nix-update-script,
+}:
+buildGoModule rec {
+  pname = "fusion";
+  version = "0.8.9";
+
+  src = fetchFromGitHub {
+    owner = "0x2E";
+    repo = "fusion";
+    rev = "v${version}";
+    hash = "sha256-nI587lshHlwZMnGGtzkLSaWc8OvY8QfPsdmAR2j+slI=";
+  };
+
+  frontend = buildNpmPackage {
```

Comment:
This should probably be in a let in

Reply:
I think you raised this point in another PR before where @Scrumplex brought up the fact that this might need to be easily overridden and so shouldn't be in a let-in. I personally don't really care about this, but if you really care, you can change every instance of this happening in Nixpkgs (even just the specific usage of `frontend = buildNpmPackage` yields 13 hits in my local repo)